data/mods/ssb/abilities.ts

```diff
@@ -1972,6 +1972,40 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		gen: 8,
 	},
 
+	// Volco
+	speedrunning: {
+		desc: "This Pokemon's speed stat is raised by 1 stage if it attacks and knocks out another Pokemon. Moves that are 60 Base Power or lower gain an additional 25 Base Power. Freezes can only be broken out of naturally, no moves can defrost a frozen Pokemon while this one is active.",
+		shortDesc: "Speed +1 on kill, Weak moves get a 25 Base Power buff. Moves cannot defrost, only natural thaws.",
+		onSourceAfterFaint(length, target, source, effect) {
+			if (effect && effect.effectType === 'Move') {
+				this.boost({spe: 1}, source);
+			}
+		},
+		onModifyMove(move, pokemon) {
+			if (move.thawsTarget) {
+				delete move.thawsTarget;
+			}
+			if (move.flags["defrost"]) {
+				delete move.flags["defrost"];
+			}
+		},
+		onFoeModifyMove(move, pokemon) {
+			if (move.thawsTarget) {
+				delete move.thawsTarget;
+			}
+			if (move.flags["defrost"]) {
+				delete move.flags["defrost"];
+			}
+		},
```

Comment:
Couldn't you merge this to `onAnyModifyMove`?

Reply:
i forgot that existed